hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -447,7 +447,7 @@ vlapic_esr_write_handler(struct acrn_vlapic *vlapic)
 {
 	struct lapic_regs *lapic;
 
-	lapic = vlapic->apic_page;
+	lapic = &vlapic->apic_page;
```

Comment:
do we need to use   &(vlapic->apic_page)?   check with Junjie.

Reply:
The code without parentheses can seem confusing to me if someone is not clear on the precedence of & and ->. However MISRA is neutral on using the parentheses here and we don't have any static checker that report on their missing. How about we make it an extra rule in the coding guideline for ACRN and I go find some alternatives that help us looking for any missed parenthesis?

Reply:
Agree! Yonghua will be the 1st to start this rule. And we will enforce later on.

Reply:
@junjiemao1 @dongyaozu 
Thanks for comments, will update following this rule.